packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -7,8 +7,24 @@ import strategies from './feedback/strategies'
 const FeedbackStore = types
   .model('FeedbackStore', {
     isActive: types.optional(types.boolean, false),
-    rules: types.map(types.frozen({}))
+    rules: types.map(types.frozen({})),
+    showModal: types.optional(types.boolean, false)
   })
+  .views(self => ({
+    get messages () {
+      const messages = []
+      self.rules.forEach(([item]) => {
```

Comment:
```suggestion
      return self.rules.map((rule) => {
        let message = false
        if (rule.success && rule.successEnabled) {
          message = rule.successMessage
        } else if (!rule.success && rule.failureEnabled) {
          message = rule.failureMessage
        }
        return message
      })
```

Reply:
Also, a note on naming: if your array is called `rules`, then members of that array should be named `rule` in the code.